dali/operators/ssd/random_crop.h

```diff
@@ -38,13 +38,13 @@ class SSDRandomCrop : public Operator<Backend> {
     int_dis_(0, 6),        // sample option
     float_dis_(0.3, 1.) {  // w, h generation
     // setup all possible sample types
-    sample_options_.push_back(SampleOption{false, -1.f});
-    sample_options_.push_back(SampleOption{false, 0.1});
-    sample_options_.push_back(SampleOption{false, 0.3});
-    sample_options_.push_back(SampleOption{false, 0.5});
-    sample_options_.push_back(SampleOption{false, 0.7});
-    sample_options_.push_back(SampleOption{false, 0.9});
-    sample_options_.push_back(SampleOption{true, 0});
+    sample_options_.emplace_back(false, -1.f);
+    sample_options_.emplace_back(false, 0.1);
+    sample_options_.emplace_back(false, 0.3);
+    sample_options_.emplace_back(false, 0.5);
+    sample_options_.emplace_back(false, 0.7);
+    sample_options_.emplace_back(false, 0.9);
+    sample_options_.emplace_back(true, 0);
```

Comment:
I think the code is equivalent (technically there's an additional move, but the object is small and trivially copyable/movable) and with the old code it was obvious to the reader that what kind of object is being added - now you have to look up the definition of `sample_options_` to see that these are `SampleOption` and not `std::pair<bool , float>`, for example.